### PR TITLE
Feat: Enable webpack caching

### PR DESCRIPTION
**Pull request by bsokol-wl**
Enables Webpack in-memory caching for faster rebuilds.

### CODE REVIEW

File: packages/webpack-configs/src/defineConfig.ts [hunk 1]
```diff
@@ -168,7 +168,7 @@ export function defineDevHostConfig(swcConfig: SwcConfig, applicationName: strin
     const {
         entry = path.resolve("./src/index.ts"),
         publicPath = "auto",
-        cache = false,
+        cache = true,
```
Comment (patricklafrance) [inline packages/webpack-configs/src/defineConfig.ts:171]: It's true by default so you don't need to specifically specify a value.

File: packages/webpack-configs/src/defineConfig.ts [hunk 2]
```diff
@@ -292,7 +290,7 @@ export function defineDevRemoteModuleConfig(swcConfig: SwcConfig, applicationNam
     const {
         entry = path.resolve("./src/register.tsx"),
         publicPath = "auto",
-        cache = false,
+        cache = true,
```
Comment (patricklafrance) [inline packages/webpack-configs/src/defineConfig.ts:293]: It's true by default so you don't need to specifically specify a value.

File: packages/webpack-configs/src/defineConfig.ts (hunk 1 above)
Comment (bsokol-wl) [reply packages/webpack-configs/src/defineConfig.ts:171]: Fixed

File: packages/webpack-configs/src/defineConfig.ts (hunk 2 above)
Comment (bsokol-wl) [reply packages/webpack-configs/src/defineConfig.ts]: Fixed

**Review by patricklafrance — APPROVED** [review]

**Pull request merged**